File: ProjetSocial/src/modele/Message.java
package modele;

import java.sql.Timestamp;

public class Message {
	
	protected int id;
	protected Timestamp moment;
	protected String text;
	protected int salon_id;
	protected int utilisateur_id;
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public Timestamp getMoment() {
		return moment;
	}
	public void setMoment(Timestamp moment) {
		this.moment = moment;
	}
	public String getText() {
		return text;
	}
	public void setText(String text) {
		this.text = text;
	}
	public int getSalon_id() {
		return salon_id;
	}
	public void setSalon_id(int salon_id) {
		this.salon_id = salon_id;
	}
	public int getUtilisateur_id() {
		return utilisateur_id;
	}
	public void setUtilisateur_id(int utilisateur_id) {
		this.utilisateur_id = utilisateur_id;
	}
}

File: Fichier PHP/model/Message.php
<?php
class Message{
    public $id;
	public $moment;
	public $text;
	public $salon_id;
    public $utilisateur_id;
    
    public function getId(){
		return $this->id;
	}
	
	public function setId($id){
		$this->id = $id;
	}
	
	public function getMoment(){
		return $this->moment;
	}
	
	public function setMoment($moment){
		$this->moment = $moment;
	}
	
	public function getText(){
		return $this->text;
	}
	
	public function setText($text){
		$this->text = $text;
	}
	
	public function getSalon_id(){
		return $this->salon_id;
	}
	
	public function setSalon_id($salon_id){
		$this->salon_id = $salon_id;
	}

	public function getUtilisateur_id(){
		return $this->utilisateur_id;
	}
	
	public function setUtilisateur_id($utilisateur_id){
		$this->utilisateur_id = $utilisateur_id;
	}
}
?>